src/routers/user.js

```diff
@@ -5,7 +5,7 @@ const router = express.Router();
 const userController = require('../controllers/user');
 
 // GET and POST routes for /api/users
-router.route('/').get(userController.getAllUsers).post(userController.addUser);
+router.route('/').get(userController.getAllUsers);
```

Comment:
You can make it even simpler like > router.get('/', userController.getAllUsers); as there are no other methods.